x/leverage/types/params.go

```diff
@@ -67,3 +83,30 @@ func validateInterestEpoch(i interface{}) error {
 	}
 	return nil
 }
+
+func validateLiquidationThreshold(i interface{}) error {
```

Comment:
Is there no maximum value?

Reply:
There's no hard value at which it becomes invalid, though it would be unreasonable to put it above, say, `0.2`

Since that's more of a soft cutoff than a "math breaks at param=x" one, I left that out of the validator.